client/src/components/Footer.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+function Footer() {
+  return (
+    <footer className="footer">
+      <h3>Designed by Bahare Bahreinian</h3>
```

Comment:
This could probably just be a `p` as there's no content under it, so it's not a subheading